prow/prowjobs/GoogleCloudPlatform/louhi/louhi-echo-test.yaml

```diff
@@ -0,0 +1,19 @@
+periodics:
+- cron: "* * 30 02 *" # Invalid date, should only run when triggered
+  name: louhi-echo-test-periodic
+  decorate: true
+  spec:
+    containers:
+    - image: alpine
+      command: ["/bin/date"]
+postsubmits:
+  https://cloud-release-internal-review.googlesource.com/louhi/test-prow:
```

Comment:
let's try it out in my personal dev repo, replacing this with `chaotoppicks/great-tools`

Reply:
Will get rid of postsubmit jobs